saleor/plugins/manager.py

```diff
@@ -532,6 +532,13 @@ def authenticate_user(self, request: WSGIRequest) -> Optional["User"]:
         default_value = None
         return self.__run_method_on_plugins("authenticate_user", default_value, request)
 
+    def external_logout(self, data: dict, request: WSGIRequest) -> dict:
+        """Authenticate user which should be assigned to the request."""
+        default_value = {}  # type: ignore
```

Comment:
Why `type: ignore` here?

Reply:
Mypy requires to provide the exact type of the dict, which plugin will return.  I can try to add some typing here.